src/shortcuts/plugin.ts

```diff
@@ -245,6 +245,16 @@ const SHORTCUTS = [
     selector: '.jp-Console .jp-CellEditor',
     sequence: ['Escape']
   },
+  {
+    command: 'terminal:increase-font',
+    selector: '.jp-TerminalWidget .jp-TerminalWidget-body',
+    sequence: ['Ctrl =']
```

Comment:
`Accel Shift =` is effectively `Cmd +`.

Implementing keyboard shortcuts in a browser is wonky, because the browser doesn't have a real keyboard API.

In brief, a shortcut is a combination of modifiers: `Alt Shift Cmd Ctrl` and primary "keycap", which is the primary silk-screened character on your keyboard. Depending on your keyboard layout, you may have `+` as a keycap, or you may not. On US-English keyboards, the `=` is the primary keycap and the `+` is the secondary keycap. 

If you don't have a US-English keyboard layout, then you will need to change the default layout on the shortcut registry. Assuming you do have US-English keyboard, here's the valid keycaps:

https://github.com/phosphorjs/phosphor/blob/master/src/ui/keyboard.ts#L183-L277